tests/acceptance/test_mender-connect.py

```diff
@@ -193,14 +202,18 @@ def test_mender_connect_auth_changes(
         """Test that mender-connect can re-establish the connection on D-Bus signals"""
 
         try:
-            dbus_set_token_and_url(connection, "token1", "http://localhost:5000")
-
             # start the mender-connect service
-            startup_time = qemu_system_time(connection)
```

Comment:
Careful :warning: By removing this we are passing to `wait_for_string_in_log` an undefined variable and the method will misbehave. We should restore it, why did you remove it in the first place?